src/commands/deploy.ts

```diff
@@ -233,6 +233,17 @@ export default class Deploy extends DeployCommand {
     const account = await AccountUtils.getAccount(this.app, flags.account);
     const get_environment_options: GetEnvironmentOptions = { environment_name: flags.environment };
     const environment = await EnvironmentUtils.getEnvironment(this.app.api, account, get_environment_options);
+    const has_cluster_deployment = await DeployUtils.hasClusterDeployment(this.app, environment.cluster);
+    if (!has_cluster_deployment) {
```

Comment:
What are your thoughts on printing a yellow warning rather than prompting?

Reply:
The reason why I did a prompt instead of a yellow warning is because the deployment will still fail with a yellow warning. But with that said, I also I'm not a big fan of the prompt either so we can change that if you want.

Reply:
Yeah I don't think that it should be blocking. If there's already a yellow warning, what are your thoughts on the detail for this ticket being added to the existing one?

Reply:
What about the following yellow warning?
```
We detected that required applications aren't installed in your cluster and this deployment will not succeed. Please stop the deployment and install the necessary applications at ${cluster_apps_url} before attempting to deploy.
```

Reply:
What about the following yellow warning?
```
We detected that required applications aren't installed in your cluster and this deployment will not succeed. Please stop the deployment and install the necessary applications at ${cluster_apps_url} before attempting to deploy.
```

Reply:
Maybe instead of "Please stop the deployment", "Please cancel the deployment"? Also I think we should name the specific cluster apps to install since that might not be clear to new users

Reply:
I have updated it to a warning and currently, it is the only warning in `runRemote`.